inductiva/logs/task_logs.py

```diff
@@ -0,0 +1,68 @@
+"""Module for streaming the logs of a task through a websocket."""
+import sys
+
+import websocket
+import logging
+
+from inductiva import constants
+
+logger = logging.getLogger("websocket")
+logger.setLevel(logging.INFO)
+
+
+class TaskLogsStream:
+    """Logs the streams of a running task through a websocket."""
+
+    def __init__(self,
+                 task_id,
+                 reconnect=5,
+                 ping_interval=15,
+                 ping_timeout=5,
+                 fout=sys.stdout,
+                 ferr=sys.stderr):
+        """Initialize websocket connection to the task STDOUT & STDERR streams.
+
+        Args:
+            task_id (int): ID of the task for which to get the STDOUT & STDERR
+                streams.
+        """
+
+        self.websocket_url = f"{constants.LOGS_WEBSOCKET_URL}/{task_id}"
+        self.task_id = task_id
+        self.reconnect = reconnect
+        self.ping_interval = ping_interval
+        self.ping_timeout = ping_timeout
+        self.fout = fout
+        self.ferr = ferr
+        self.ws = self.setup_websocket()
+
+    def setup_websocket(self):
```

Comment:
this method can also be made private.
By making these methods private, we just need to document them from the point of view of the developer and not the user.